osu.Game/Overlays/UserProfileOverlay.cs

```diff
@@ -73,16 +73,15 @@ protected override void PopOut()
             FadeEdgeEffectTo(0, WaveContainer.DISAPPEAR_DURATION, Easing.Out);
         }
 
-        public void ShowUser(long userId)
-        {
-            if (userId == Header.User.Id)
-                return;
-
-            ShowUser(new User { Id = userId });
-        }
+        public void ShowUser(long userId) => ShowUser(new User { Id = userId });
 
         public void ShowUser(User user, bool fetchOnline = true)
         {
+            Show();
+
+            if (user.Id == Header?.User.Id)
```

Comment:
The null check here should also resolve https://sentry.io/share/issue/0de998bd00b2417fa61f78f9c0cc47ed/